go.mod

```diff
@@ -7,21 +7,21 @@ require (
 	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
-	github.com/Microsoft/hcsshim v0.8.7 // indirect
+	github.com/Microsoft/hcsshim v0.8.9 // indirect
 	github.com/apache/thrift v0.0.0-20161221203622-b2a4d4ae21c7 // indirect
 	github.com/codahale/hdrhistogram v0.0.0-20160425231609-f8ad88b59a58 // indirect
-	github.com/containerd/cgroups v0.0.0-20200217135630-d732e370d46d // indirect
-	github.com/containerd/console v0.0.0-20191219165238-8375c3424e4d
+	github.com/containerd/cgroups v0.0.0-20200327175542-b44481373989 // indirect
+	github.com/containerd/console v1.0.0
 	github.com/containerd/containerd v1.4.0-0
-	github.com/containerd/continuity v0.0.0-20200107194136-26c1120b8d41
-	github.com/containerd/fifo v0.0.0-20191213151349-ff969a566b00 // indirect
+	github.com/containerd/continuity v0.0.0-20200413184840-d3ef23f19fbb
```

Comment:
does not seem to match https://github.com/containerd/containerd/blob/04985039cede6aafbb7dfb3206c9c4d04e2f924d/vendor.conf#L7

Reply:
oh, hmf; I think I've been a bit ahead; had this branch open locally; https://github.com/containerd/containerd/pull/4247

let me know if you want to revert

Reply:
I assume you want these to get to moby so you can just wait until contained pr gets merged and then update this.

Reply:
Yeah, both moby and containerd are pending a new release, so want to have dependencies up to date (and cleaned up where possible)